src/Raven.Server/Web/System/Processors/Backups/BackupDatabaseHandlerProcessorForGetPeriodicBackupStatus.cs

```diff
@@ -25,13 +27,46 @@ public override async ValueTask ExecuteAsync()
 
         using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
         using (context.OpenReadTransaction())
-        using (var statusBlittable = ServerStore.Cluster.Read(context, PeriodicBackupStatus.GenerateItemName(name, taskId.Value)))
-        await using (var writer = new AsyncBlittableJsonTextWriter(context, RequestHandler.ResponseBodyStream()))
         {
-            writer.WriteStartObject();
-            writer.WritePropertyName(nameof(GetPeriodicBackupStatusOperationResult.Status));
-            writer.WriteObject(statusBlittable);
-            writer.WriteEndObject();
+            var dbRecord = RequestHandler.ServerStore.Cluster.ReadRawDatabaseRecord(context, name);
+            if (dbRecord.IsSharded == false)
+            {
+                using (var statusBlittable = ServerStore.Cluster.Read(context, PeriodicBackupStatus.GenerateItemName(name, taskId.Value)))
+                await using (var writer = new AsyncBlittableJsonTextWriter(context, RequestHandler.ResponseBodyStream()))
+                {
+                    writer.WriteStartObject();
+                    writer.WritePropertyName(nameof(GetPeriodicBackupStatusOperationResult.IsSharded));
+                    writer.WriteBool(false);
+                    writer.WriteComma();
+                    writer.WritePropertyName(nameof(GetPeriodicBackupStatusOperationResult.Status));
+                    writer.WriteObject(statusBlittable);
+                    writer.WriteEndObject();
```

Comment:
instead of this, can we make an object for the result with a `ToJson`?

Reply:
changed to have a single `DynamicJsonValue` result